pages/linux/terminator.md

```diff
@@ -0,0 +1,19 @@
+# terminator
+
+> Multiple GNOME terminals in one window.
+
+- Start with a fullscreen window:
```

Comment:
We usually start with the simplest command invocation - i.e. `terminator` and explain what it does.

Reply:
👍